### PR TITLE
Gitblit groovy script updates:

### PR DESCRIPTION
**Pull request by edagan**
1. Renamed to verigreen.groovy due to groovy issue running script with hyphens in their name
2. Windows support enhancements
3. Log improvements

### CODE REVIEW

File: utils/hooks/gitblit-only/verigreen.groovy [hunk 1]
```diff
@@ -35,11 +35,16 @@ import org.slf4j.Logger
 * Globals
 *********************/
 // Change these as needed:
-@Field def JAVA_HOME = '/usr/lib/jvm/java-7-oracle'               // leave empty if already configured in environment
-@Field def VG_HOOK   = '/opt/gitblit-data/groovy/vg-hook-2.0.1'  // path to hook.properties file
-@Field def VG_PATH   = '/opt/gitblit-data/groovy/vg-hook-2.0.1' // path to git-hook.jar file
+// Windows only: In case JAVA_HOME path contains spaces, encapsulate it with quotes 
+@Field def JAVA_HOME = '"c:/Program Files/Java/jre7"'               // leave empty if already configured in environment
+// VG paths must NOT be encapsulates with quotes!
```
Comment (eitansch) [inline utils/hooks/gitblit-only/verigreen.groovy]: encapsulateD. But better yet - escaped :-)

File: utils/hooks/gitblit-only/verigreen.groovy [hunk 2]
```diff
@@ -64,6 +69,8 @@ private def executeOnShell(String command, File workingDir, StringBuilder output
     if (!JAVA_HOME.equals("")) {     // use the JAVA_HOME if it is defined (earlier in this script)
         env.put("JAVA_HOME", JAVA_HOME)
     }
+	logger.info("VG_HOOK:" + VG_HOOK + ln )
```
Comment (eitansch) [inline utils/hooks/gitblit-only/verigreen.groovy]: Do you need those for production?

File: utils/hooks/gitblit-only/verigreen.groovy [hunk 3]
```diff
@@ -111,6 +119,7 @@ for (ReceiveCommand command : commands) {
             command.getOldId().name() + " " +
             command.getNewId().name() + " " +
             command.refName
+	logger.info('vgCmd: ' + vgCmd.toString())
```
Comment (eitansch) [inline utils/hooks/gitblit-only/verigreen.groovy]: Needed for production?

File: utils/hooks/gitblit-only/verigreen.groovy (hunk 2 above)
Comment (edagan) [inline utils/hooks/gitblit-only/verigreen.groovy]: Not all messages are needed for production, I'll get to adding a variable to control this

Eli

File: utils/hooks/gitblit-only/verigreen.groovy [hunk 4]
```diff
@@ -35,11 +35,16 @@ import org.slf4j.Logger
 * Globals
 *********************/
 // Change these as needed:
-@Field def JAVA_HOME = '/usr/lib/jvm/java-7-oracle'               // leave empty if already configured in environment
-@Field def VG_HOOK   = '/opt/gitblit-data/groovy/vg-hook-2.0.1'  // path to hook.properties file
-@Field def VG_PATH   = '/opt/gitblit-data/groovy/vg-hook-2.0.1' // path to git-hook.jar file
+// Windows only: In case JAVA_HOME path contains spaces, encapsulate it with quotes 
+@Field def JAVA_HOME = '"c:/Program Files/Java/jre7"'               // leave empty if already configured in environment
+// VG paths must NOT be encapsulates with quotes!
+@Field def VG_HOOK   = 'c:\\Program Files\\Apache Software Foundation\\Tomcat 7.0\\webapps\\gitblit-1.6.2\\WEB-INF\\data\\groovy\\vg'  // path to hook.properties file
```
Comment (eitansch) [inline utils/hooks/gitblit-only/verigreen.groovy:43]: Perhaps leave (for reference's sake) both options of Windows/Linux?

**Pull request closed without merging**